src/Resources/app/administration/src/snippet/en-GB.json

```diff
@@ -58,6 +58,8 @@
         "redBackgroundDescription": "The order(s) does not have a shipping method equal to InPost or the order(s) don't have created InPost package",
         "greenBackgroundTitle": "Green background",
         "greenBackgroundDescription": "A courier for this package(s) has already been ordered",
+        "yellowBackgroundTitle": "Yellow background",
+        "yellowBackgroundDescription": "The order(s) have a shipping method equal to InPost, but the package has selected parcel locker as sending method",
```

Comment:
This and the one above about red background should be rephrased to something along the lines of:

The order's shipping method is InPost but the package's sending method is set to parcel locker.

Reply:
Changed in next commit